### ISSUE TITLE
Global Road Damage Detection Challenge 2020

### ISSUE THREAD

**Issue by Big-Monkey-O** [sekilab/RoadDamageDetector#43]
Hello, why can’t I access the website of the IEEE-Big-Data 2020 Road Damage Challenge? Is the server being maintained? Or do you plan to close this website?

**Comment by Archer-pro666** [comment]
I also encountered this problem

**Comment by Archer-pro666** [comment]
Now the website can be accessed

**Comment by Big-Monkey-O** [comment]
> Now the website can be accessed

Ok, thank you for your information

**Comment by wendysnowhehe** [comment]
Hey guys! Could u log in on the website now?

**Comment by Big-Monkey-O** [comment]
No，I can't log in on the website




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "sekilab/RoadDamageDetector"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月11日(星期二) 下午4:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [sekilab/RoadDamageDetector] Global Road Damage Detection Challenge 2020 (Issue #43)





 
Hey guys! Could u log in on the website now?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wendysnowhehe** [comment]
I'm sorry to hear that. Are you still working on this data set?

**Comment by Big-Monkey-O** [comment]
Yes, I divide a data set from the training data set to do my experiment




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "sekilab/RoadDamageDetector"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月11日(星期二) 晚上11:06
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [sekilab/RoadDamageDetector] Global Road Damage Detection Challenge 2020 (Issue #43)





 
I'm sorry to hear that. Are you still working on this data set?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wendysnowhehe** [comment]
Yeah, I'm gonna do that, too.



| |
贺雪洁
|
|
***@***.***
|
签名由网易邮箱大师定制
On 1/11/2022 ***@***.***> wrote：

Yes, I divide a data set from the training data set to do my experiment




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "sekilab/RoadDamageDetector" ***@***.***&gt;;
发送时间:&nbsp;2022年1月11日(星期二) 晚上11:06
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [sekilab/RoadDamageDetector] Global Road Damage Detection Challenge 2020 (Issue #43)






I'm sorry to hear that. Are you still working on this data set?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wendysnowhehe** [comment]
Yeah, I'm gonna do that, too.

**Comment by Archer-pro666** [comment]
I can't log in either, I thought it was just me。。。

**Comment by wendysnowhehe** [comment]
It's not just you.  I also want to ask, did you just use this one data set or did you merge 2019 and 2020?

**Comment by Archer-pro666** [comment]
> 

OK, I just use this one data set.
